src/pages/code-intelligence-platform.tsx

```diff
@@ -0,0 +1,388 @@
+import { FunctionComponent, ReactNode } from 'react'
+
+import ChartBoxIcon from 'mdi-react/ChartBoxIcon'
+import ClipboardCheckIcon from 'mdi-react/ClipboardCheckIcon'
+import EyeIcon from 'mdi-react/EyeIcon'
+import ReiterateIcon from 'mdi-react/ReiterateIcon'
+import ViewDashboardIcon from 'mdi-react/ViewDashboardIcon'
+import Link from 'next/link'
+
+import {
+    Layout,
+    BlogResourceItem,
+    ContentSection,
+    CustomCarousel,
+    TwoColumnSection,
+    QuoteCarousel,
+    Video,
+} from '@components'
+import { buttonStyle, buttonLocation } from '@data'
+
+const CarouselItem: FunctionComponent<{ icon: ReactNode; header: string; text: ReactNode }> = ({
+    icon,
+    header,
+    text,
+}) => (
+    <>
+        {icon}
+        <h3 className="font-weight-bold mb-lg-0 mb-5">{header}</h3>
+        {text}
+    </>
+)
+
+const items = [
+    {
+        title: 'Full view of the codebase',
+        subtitle: 'Exhaustive across all repositories and code hosts',
+        text: (
+            <CarouselItem
+                icon={
+                    <div className="bg-violet-1 rounded-circle max-w-50 text-center p-1 mb-3">
+                        <EyeIcon size={40} className="p-1 text-vivid-violet" />
+                    </div>
+                }
+                header="Full view of the codebase"
+                text={
+                    <p className="py-3">
+                        High-fidelity search is applied across all repositories and code hosts to provide a holistic
+                        view of sprawling code. Developers are in control of how they prefer to navigate the codebase
+                        with multiple pattern types, including symbol search, structural search, and precise code
+                        navigation.
+                    </p>
```

Comment:
Good idea. To be able to pass the icon pathname more dynamically I think we need a different icon library, though. I'm not seeing the syntax for [@mdi/react](https://www.npmjs.com/package/@mdi/react). I brought in [mdi-react](https://www.npmjs.com/package/mdi-react) to do this better, looks like this is what sourcegraph/sourcegraph uses. Maybe this will be a better alternative to help us with our icon re-design & refactoring (#5432). What do you think @bretthayes ?

Reply:
Can you look at the impact this has on bundle size? I'd probably want to introduce a new icon library carefully in another ticket/pr and then do #5432 across the codebase because we had some issues when we tried that and it slowed down performance. But I do like that you can do multiple imports with destructuring assignment 🤓 

With my previous mention, you can just reference the icon as mentioned `icon={EyeIcon}` and then in your loop just do `<icon>` and it should render.

Reply:
I tried that before and it's not working. Do you know the syntax? I can't find anything. This commit has your suggested setup [f15fc53](https://github.com/sourcegraph/about/pull/5641/commits/f15fc5362fcdfaa19e9c9418e1d01d504e24779c)

Reply:
No worries! I pushed up a commit for you to fix icon syntax, but your text content isn't rendering so you'll have to check that out

Reply:
Thanks! I had to add a few things for styling too, but got it up on [06cca4d](https://github.com/sourcegraph/about/pull/5641/commits/06cca4de6e1e3ceab4ac0c94a74b2ccf8f3728bb). I wish React could do dynamic components as easily as [Vue does](https://v2.vuejs.org/v2/guide/components-dynamic-async.html), they write it as simple as `<Component is={passedComponent} />`, that way we can keep the `size` and `class` properties more efficiently